src/VirtualClient/VirtualClient.Common/ProcessDetails.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace VirtualClient.Common
+{
+    /// <summary>
+    /// Process details.
+    /// </summary>
+    public class ProcessDetails
+    {
+        /// <summary>
+        /// Id of the process that ran.
+        /// </summary>
+        public int Id { get; set; }
+        
+        /// <summary>
+        /// Command line executed.
+        /// </summary>
+        public string CommandLine { get; set; }
+
+        /// <summary>
+        /// Exit code of the command executed.
+        /// </summary>
+        public int ExitCode { get; set; }
+
+        /// <summary>
+        /// Generated Results of the command.
+        /// </summary>
+        public string GeneratedResults { get; set; }
```

Comment:
Need to support more than 1 set of results. Some of the workloads produce multiple files worth of output for a single run of the process.